### PR TITLE
feat: added php 8 support

### PR DESCRIPTION
**Pull request by bricejulia**

### CODE REVIEW

**Comment by bpolaszek** [comment]
ping @dunglas @Toflar - I know you guys are probably busy but it would be great if you could review that one 🙂

File: composer.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
     }
   ],
   "require": {
-    "php": "^7.1",
+    "php": "^7.1 || ^8.0",
```
Comment (dunglas) [inline composer.json:16]: We could even could go with `>=7.1`!

**Review by dunglas — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (bricejulia) [reply composer.json]: Done !

**Comment by dunglas** [comment]
Thanks @bricejulia!

**Comment by gveliberty** [comment]
@dunglas @Toflar the travis build failed on what seems to be a temporary error : 
```
$ git fetch origin +refs/pull/92/merge:
fatal: couldn't find remote ref refs/pull/92/merge
```
Could you please re-run it :pray: 
I'm sure a lot of us are waiting for a new release.

**Comment by Toflar** [comment]
https://github.com/dunglas/doctrine-json-odm/releases/tag/1.1.0 😊

**Comment by bpolaszek** [comment]
🚀  Thanks to everyone involved! 🙏

**Pull request merged**